Data/member.json

```diff
@@ -414,5 +414,12 @@
             "githuburl": "https://github.com/ZuhairHossain/",
             "linkedinurl": "https://www.linkedin.com/in/syed-zuhair-hossain/"
         }
```

Comment:
@abhyuday26 so the file makes sense, is it possible for you to add a comma after the brace? Thank you.